src/Components/Content.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+
+const Content = () => {
+
+    const [reviews, setReviews] = useState([])
+
+    function getReviews() {
+
+        axios.get('https://ikrams-first-app.herokuapp.com/api/reviews')
+            .then((response) => {
+                setReviews(response.data.reviews)
+            })
+            .catch((err) => {
+                console.log(err)
+            })
+    }
+
```

Comment:
It would be better and easier if we abstract all the api calls into another file and import it in each component